Jason/playlist_challenge.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
```

Comment:
Good alias - can you think of an example where an alias could be beneficial and an example of when it can make coding worse?